pinot-transport/src/test/java/com/linkedin/pinot/transport/common/routing/RoutingTableTest.java

```diff
@@ -400,4 +409,95 @@ public void testCombinedKafkaRouting() throws Exception {
     }
     return configs;
   }
+
+  class FakePropertyStore extends ZkHelixPropertyStore<ZNRecord> {
```

Comment:
It's not unfortunately (it just stores one znrecord and always returns the same one, ignoring the path argument), though it would be really handy to have a one that's generic enough to be reused.